files/ko/web/javascript/reference/global_objects/object/getownpropertynames/index.html

```diff
@@ -0,0 +1,151 @@
+---
+title: Object.getOwnPropertyNames()
+slug: Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames
+tags:
+  - ECMAScript 5
+  - JavaScript
+  - JavaScript 1.8.5
+  - Method
+  - Object
+  - Reference
+  - Polyfill
+browser-compat: javascript.builtins.Object.getOwnPropertyNames
+---
+<div>{{JSRef}}</div>
+
+<p><strong><code>Object.getOwnPropertyNames()</code></strong> 메서드는 전달된 객체의 모든 속성 (심볼을 사용하는 속성을 제외한 열거할 수 없는 속성 포함) 들을 배열로 반환합니다.</p>
+
+<div>{{EmbedInteractiveExample("pages/js/object-getownpropertynames.html")}}</div>
+
+<h2 id="Syntax">구문</h2>
+
+<pre class="brush: js">Object.getOwnPropertyNames(<var>obj</var>)</pre>
+
+<h3 id="Parameters">매개변수</h3>
+
+<dl>
+ <dt><code><var>obj</var></code></dt>
+ <dd>반환 받을 열거형 속성과 열거형이 아닌 속성을 가진 객체</dd>
+</dl>
+
+<h3 id="Return_value">반환 값</h3>
+
+<p>전달된 객체에 있는 속성들의 문자열 배열을 반환합니다.</p>
+
+<h2 id="Description">설명</h2>
+
+<p><code>Object.getOwnPropertyNames()</code> 는 전달된 객체(<code><var>obj</var></code>)의 열거형 및 열거할 수 없는 속성들을 문자열 배열로 반환합니다.
+    배열의 열거할 수 있는 속성들의 순서는 {{jsxref("Statements/for...in", "for...in")}} 반복문 (또는 {{jsxref("Object.keys()")}})의 결과 값과 일치합니다.
```

Comment:
```suggestion
    배열의 열거할 수 있는 속성들의 순서는 {{jsxref("Statements/for...in", "for...in")}} 반복문 (또는 {{jsxref("Object.keys()")}})이 처리되는 순서와 일치합니다.
```
"결과" 보단 "순서"가 더 원문 의미와 맞는 것 같습니다.